includes/snmp.inc.php

```diff
@@ -267,7 +267,7 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
     $time_start = microtime(true);
 
     if (strstr($oid, ' ')) {
-        echo report_this_text("snmp_get called for multiple OIDs: $oid");
+        echo "snmp_get called for multiple OIDs: $oid";
```

Comment:
This should be `throw new Exception("snmp_get called for multiple OIDs: $oid")`

Reply:
This was fixed in https://github.com/librenms/librenms/pull/10728